contracts/contracts/UmpireRegistry.sol

```diff
@@ -24,25 +22,31 @@ contract UmpireRegistry is KeeperCompatibleInterface, Ownable {
     }
 
     function createJobFromNodes(
+        string memory _name,
         PostfixNode[] memory _postfixNodesLeft,
         UmpireComparator _comparator,
         PostfixNode[] memory _postfixNodesRight,
         address[] memory _dataFeeds,
-        uint _timeout,
+        uint _activationDate,
+        uint _timeoutDate,
         address _action
     ) external returns (uint jobId) {
-        require(_timeout > block.timestamp + 5 minutes, "Timeout at least 5 minutes into the future is required");
+        require(_timeoutDate > block.timestamp + 5 minutes, "Timeout date at least 5 minutes into the future is required");
```

Comment:
do we want to set those 5 and 1 minutes as something configurable by the owner?

Reply:
Maybe would make sense to make it configurable by the registry owner (so in an ideal world a DAO), the basic idea behind those constraints is that creating a job that would instantly complete (fail or succeed) in the first block makes no sense, there's no time for the job creator to set up the action etc. This whole system isn't really usable to the "flashbots" because it relies on Chainlink Automation, which doesn't guarantee execution (performing upkeep) in the same block that the condition is met. So I put in those constraints to prevent users from accidentally setting up a job that would auto-fail for example. But now that I think about it, 5 minutes is way too long, I'll make it configurable later on, thanks.